tests/ImageSharp.Tests/Drawing/DrawPathTests.cs

```diff
@@ -32,13 +33,13 @@ public void ImageShouldBeOverlayedByPath()
                     new Vector2(60, 10),
                     new Vector2(10, 400));
 
-                CorePath p = new CorePath(linerSegemnt, bazierSegment);
+            	ShapePath p = new ShapePath(linerSegemnt, bazierSegment);
```

Comment:
If they want to do complex paths featuring both linear & bezier segments they will do... the shapes library does have a shape builder class that can be used to declarativly draw complex combinations of paths and shapes which I imagine would end up being the preferred way of creating large complex shapes that they can then cache and redraw many times.